packages/visx-shape/src/shapes/Arc.tsx

```diff
@@ -41,13 +38,12 @@ export default function Arc<Datum>({
 
   // eslint-disable-next-line react/jsx-no-useless-fragment
   if (children) return <>{children({ path })}</>;
-  if (!data) return null;
```

Comment:
since `data` is already optional, I think this is probably the only line that needs to be removed since it is valid to call `path` without arguments.